src/vscode-azurefunctions.api.d.ts

```diff
@@ -78,4 +79,9 @@ export interface ICreateFunctionOptions {
      * If set to true, it will not try to open the folder after create finishes. Defaults to false
      */
     suppressOpenFolder?: boolean;
+
+    /**
+     * If set, it will automatically select the worker runtime for .NET with the matching targetNetwork
+     */
+    targetNetwork?: TargetFramework;
```

Comment:
I don't think I want to start hard-coding target framework versions in the api. Can we just leave this as `string`?